gel/datatypes/object.c

```diff
@@ -284,7 +265,7 @@ object_repr(EdgeObject *o)
 
     if (_EdgeGeneric_RenderItems(&writer,
                                  (PyObject *)o, o->desc,
-                                 o->ob_item, Py_SIZE(o), 1, 0) < 0)
+                                 o->ob_item, Py_SIZE(o), 1, 1, 0) < 0)
```

Comment:
You don't really need to fix this now, but this sort of list of boolean arguments ought to either have comments saying what each of the argument names are or should use #defines/enums or flag bits or something. `1, 1, 0` is inscrutable.

This is an existing problem, though, so it's fine to skip fixing it now.

Reply:
Fixed in [Apply better style for passing flags](https://github.com/edgedb/edgedb-python/pull/560/commits/c62afea58c1bcb46759038e2354d1437ea20dd8b)

Reply:
Much nicer